### ISSUE TITLE
error occurred

### ISSUE THREAD

**Issue by ForrestPi** [cpacker/MemGPT#670]
**Describe the bug**

**memgpt/autogen/memgpt_agent.py", line 216
    match type:
          ^
SyntaxError: invalid syntax**

- [ ] MemGPT version
  - pymemgpt==0.2.7
- [ ] How did you install memgpt?
  - `pip install pymemgpt`
- [ ] Describe your setup
  - Linux
  - Terminal

**Comment by cpacker** [comment]
can you add your autogen version? `pip list | grep pyautogen`

**Comment by ForrestPi** [comment]
pyautogen  0.2.2 
Python  3.9.18

**Comment by ForrestPi** [comment]
when 
```
from memgpt.autogen import memgpt_agent as memgpt_autogen
```
```
memgpt/autogen/memgpt_agent.py", line 216
match type:
^
SyntaxError: invalid syntax
```

**Comment by ForrestPi** [comment]
@cpacker

**Comment by cpacker** [comment]
@ForrestPi this should be fixed by PR #691 (which will be part of the upcoming 0.2.10 release). Should also be fixable by upgrading your python to python >= 3.10.
